standalone-metastore/packaging/pom.xml

```diff
@@ -0,0 +1,228 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.apache.hive</groupId>
+    <artifactId>hive-standalone-metastore</artifactId>
+    <version>4.2.0-SNAPSHOT</version>
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+
+  <artifactId>metastore-packaging</artifactId>
+  <packaging>pom</packaging>
+  <name>Hive Metastore Packaging</name>
+  
+  <properties>
+    <maven.compiler.source>17</maven.compiler.source>
+    <maven.compiler.target>17</maven.compiler.target>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <standalone.metastore.path.to.root>..</standalone.metastore.path.to.root>
+    <maven.license.plugin.version>2.3.0</maven.license.plugin.version>
+  </properties>
+  <profiles>
+    <profile>
+      <id>dist</id>
+      <build>
+        <plugins>
+          <!-- plugins are always listed in sorted order by groupId, artifactId -->
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-assembly-plugin</artifactId>
+            <executions>
+              <execution>
+                <id>assemble</id>
+                <phase>package</phase>
+                <goals>
+                  <goal>single</goal>
+                </goals>
+                <configuration>
+                  <finalName>apache-${project.parent.artifactId}-server-${project.version}</finalName>
```

Comment:
do we need to keep `server`?

Reply:
I rename it to `hive-standalone-metastore-4.2.0-SNAPSHOT-bin.tar.gz` to align with the old releases.

Reply:
@dengzhhu653 , fyi that is not an archive name, but a folder inside. old release had `apache-hive-metastore-3.0.0-bin`

Reply:
This PR keeps the old name pattern without the `apache-`